metadata/lang/es.yml

```diff
@@ -4,6 +4,7 @@ questions: "Preguntas"
 objectives: "Objetivos"
 requirements: "Requisitos"
 authors: "Autores"
+add contributions: "¡Agregue contribuciones!"
```

Comment:
I am not sure if spaces will work here, can you change it to `add-contributions` in both files?

Reply:
Okay, I'll do that. Thank you❤